src/prefect/client/orchestration.py

```diff
@@ -2694,6 +2694,89 @@ async def release_concurrency_slots(
             },
         )
 
+    async def create_concurrency_limit_global(
+        self,
+        name: str,
+        limit: int,
+        active: bool = True,
+        active_slots: int = 0,
+        slot_decay_per_second: int = 0,
+    ) -> UUID:
+        response = await self._client.post(
+            "/v2/concurrency_limits/",
+            json={
```

Comment:
Generally the way we handle these create/update methods is to have a schema in `actions` that has the fields we're going to send to the server, you can see examples of the server side of this here: https://github.com/PrefectHQ/prefect/blob/main/src/prefect/server/schemas/actions.py#L438-L459

Essentially you'd want to create the same objects in `client/schemas/actions.py` and `client/schemas/objects.py` and have that be the argument you pass to the create/update/etc. method:

```python
from prefect.client.schemas.actions imoport (
    GlobalConcurrencyLimitCreate
    GlobalConcurrencyLimitUpdate
)

async def create_global_concurrency_limit(
    self,
    concurrency_limit: GlobalConcurrencyLimitCreate,
) -> UUID:
    response = await self._client.post(
        "/v2/concurrency_limits/",
        json=concurrency_limit.dict(json_compatible=True, exclude_unset=True),
    )
    return UUID(response.json()["id"])

async def update_global_concurrency_limit(
    self,
    concurrency_limit: GlobalConcurrencyLimitUpdate,
):
    ...
```

There are a few reasons for this:

1.The Pydantic model (which is what those actions are) allows you to set fields as optional, so those fields can just be skipped entirely when a user is instantiating the action, so the user only needs to specify what's needed.
2. The actions models validate typing, and can do deeper validation as well, say on the name of the concurrency limit. It's much nicer to have the validation happen before the server call, in general.
3. `exclude_unset`, this is important for the `patch` call below, if you leave `name` unset in the action the value won't get sent to the sever and the server won't change the name of the limit because it didn't get a value for the name.

Reply:
OK I will look into it. Thanks for your explanations, these will help a lot!